packages/forma-36-tokens/src/tokens/typography/font-weight.js

```diff
@@ -1,6 +1,6 @@
 const fontWeight = {
   'font-weight-normal': 400,
-  'font-weight-medium': 600,
+  'font-weight-medium': 500,
```

Comment:
This is a breaking change 🙀

Reply:
hmmmm and that's the most important part for me hahaha

@mshaaban0 @fabe and @burakukula why didn't you say in slack that we didn't have breaking changes in Typography?

What do we do now, friends?

Reply:
If we change our typography it's probably not breaking, but the token change is a breaking change IMO.

But we should align with https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping

Reply:
@denkristoffer @gui-santos change like it is done in here it's not a breaking change - we adjust the value of the token, not changing the namespace for the token itself. 
If we decide to change tokens namespaces, then yeah, it's v4, but I am not sure if we want to do it tbh. 
I am ok with this adjustment like it is for now.

Reply:
Changing the value here doesn't seem like a breaking change to me

Reply:
After giving this a second thought I agree that this is no more breaking than all the other token changes we've done recently. I feel strongly about aligning with the normal weight names though 🙂